src/mumble/Messages.cpp

```diff
@@ -59,6 +59,26 @@
 		return; \
 	}
 
+/// This message is being received after the client send a Capabilities message with its preferred protocols inside.
+/// The server replies with a Capabilities message with only one protocol in it to indicate the protocol it has chosen.
+///
+void MainWindow::msgCapabilities(const MumbleProto::Capabilities &msg) {
+	if (msg.supported_protocols_size() == 0)
+		return;
+
+	ConnectionPtr c = g.sh->cConnection;
+
+	auto protocol_type = msg.supported_protocols(0);
```

Comment:
As explained in Mumble.proto, the server returns a Capabilities packet with only one item to indicate the protocol it decided to use.

Reply:
This should be a comment in the actual file instead of a review comment :D